CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+# Contributing to elixir-nats
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved!
+
+## Using the issue tracker
+
+Use the issues tracker for:
+
+* [bug reports](#bug-reports)
+* [submitting pull requests](#pull-requests)
+
+Please **do not** use the issue tracker for personal support requests nor feature requests. Instead,
+please reach out to us at
+
+* [info@nats.io](mailto:info@nats.io)
+* [Twitter](https://twitter.com/nats_io)
+* [Google Mailing List](https://groups.google.com/forum/#!forum/natsio)
+* [Slack](https://docs.google.com/a/apcera.com/forms/d/104yA7oqq7SPoMDG_J9MnVE74gVwBnTmVHKP5ABHoM5k/viewform?embedded=true)
+
+
+## Bug reports
+
+A bug is a _demonstrable problem_ that is caused by the code in the repository.
+Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; [check if the issue has already been
+   reported](https://github.com/nats-io/elixir-nats/search?type=Issues)
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   `master` branch in the repository.
+
+3. **Isolate and report the problem** &mdash; ideally create a reduced test
+   case.
```

Comment:
How about ideally create a test case in the _test directory_ demonstrating the issue as simply and with as little extraneous code as possible.